packages/core/src/payment/strategies/braintree/braintree-payment-processor.ts

```diff
@@ -45,6 +45,17 @@ export default class BraintreePaymentProcessor {
     }
 
     async tokenizeCard(payment: OrderPaymentRequestBody, billingAddress: Address): Promise<NonceInstrument> {
+        const {
+            ccNumber,
+            ccExpiry,
+        } = payment.paymentData as CreditCardInstrument;
+
+        const isValidRequiredFields = !!(ccNumber && ccExpiry);
+
+        if (!isValidRequiredFields) {
+            throw new PaymentArgumentInvalidError();
```

Comment:
Can we use a more shopper friendly error type? For example, `PaymentInvalidFormError`.
"Unable to submit payment for the order because the payload is invalid" seems too nerdy.

Reply:
@bc-peng updated. check please